mittab/apps/tab/team_views.py

```diff
@@ -380,3 +380,79 @@ def team_stats(request, team_id):
     return JsonResponse(data)
 
 
+def public_status(request, team_id, team_ref=None):
+    matches = Team.objects.filter(pk=team_id)
+    
+    if matches:
+        round_number = TabSettings.get('cur_round') - 1        
+        pairing_exists = TabSettings.get('pairing_released', 0) == 1
+
+        if pairing_exists:
+            return _public_status_pairing_exists(request, matches, round_number)
+        else:
+            return render_to_response('public_status.html',
+                                      {
+                                        'no_navigation': True,
+                                        'bye': False,
+                                        'pairing_exists': False,
+                                        'team_name': None,
+                                        'gov_or_opp': None,
+                                        'other_team': None,
+                                        'room': None,
+                                        'judges': None,
+                                        'round_number': round_number,
+                                      }, context_instance=RequestContext(request))
```

Comment:
BTW: Can you try making these changes to all of the `render_to_response` calls:

1. Remove the context_instance keyword-arg
2. `from django.shortcuts import render` and then replace `render_to_response` with `render`

I believe this code is not compatible with django 1.10, which I'd like to upgrade soon. Example: https://github.com/MIT-Tab/mit-tab/pull/148/files

Reply:
Done!

Reply:
Awesome, thanks! Gonna run this locally over the weekend to test it out, then merge into a non-master branch so that the integration of the feature is delayed until I can re-design the homepage a bit